app/resources/app-base.html.template

```diff
@@ -14,7 +14,13 @@
 {% block title %}{{block.super}}{% endblock title %}
 
 {% block logo %}
-  {% if not env.amp %}
+  {% if env.amp %}
+    <div class="header-space">
+      <a href="/global/">
+        <amp-img width="24" height="24" src="{{env.global_url}}/back_arrow.png" alt="About Person Finder">
```

Comment:
Use {% trans "About Person Finder" %} for i18n.

Reply:
done.